config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
should this be hardcoded?

Reply:
This file has been added automatically by the update script. It contains configuration for "Action Cable", a pretty new feature using WebSockets and I'm sure we are not using it and the redis URL specified here is irrelevant.